generator/sig_readme.tmpl

```diff
@@ -10,6 +10,9 @@
 Meeting notes and Agenda can be found [here]({{.MeetingArchiveURL}}).
 Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 
+## Label
```

Comment:
I'm not sure adding this is needed in the sig readme.. doesn't add a lot of information.

Reply:
Please see the issue  #1172 by @ahmetb.  Users find sig list doesn't help /sig command  and have a confuse which are valid value in commond as following:

When the k8s-merge-bot tells me to add a /sig label (example), It redirects me to this list: https://github.com/kubernetes/community/blob/master/sig-list.md

But this list is all human-readable names e.g. "Contributor Experience" but I can't use this in /sig command. It's unclear to find out which one is the correct value:
I had to visit https://github.com/kubernetes/kubernetes/labels to find out which values are valid.

Reply:
I think @cblecker meant that it's not so necessarily to add label to the README file of each sig.
Just add a `Label` column to sig-list.md (as you already did in this PR) to indicate the sig labels would be enough.

Reply:
Yes, thanks for clarifying @xiangpengzhao. That's what I meant. Putting the label into the sig-list.md template is good, but I don't think we need a separate section in every README.md with just the label.

Reply:
@cblecker What do I do for it?  I have to add label into sig.yml and add lable field into Group in app.go. Then exectue the "make" command and all README.md were added label.

Reply:
OK, I got it and update